types/nightwatch/index.d.ts

```diff
@@ -1698,7 +1698,7 @@ export interface Level {
     /**
```

Comment:
In `Level` interface `name` should be change from `string` to `'ALL' | 'DEBUG' | 'FINE' | 'FINER' | 'FINEST' | 'INFO' | 'OFF' | 'SEVERE' | 'WARNING'`